Source/WebInspectorUI/UserInterface/Controllers/NetworkManager.js

```diff
@@ -1561,6 +1569,7 @@ WI.NetworkManager = class NetworkManager extends WI.Object
     _sourceMapLoadAndParseFailed(sourceMapURL)
     {
         this._downloadingSourceMaps.delete(sourceMapURL);
+        this._failedSourceMapURLs.add(sourceMapURL);
```

Comment:
i think there should also be a `this._failedSourceMapURLs.clear();` in `_handleFrameMainResourceDidChange`, as otherwise we will always consider any URL within to be a failed sourcemap load even if it does succeed after the inspected page navigates (e.g. refreshing the page after connecting to VPN, adjust the inspected page URL to include a secret key, etc.)

Reply:
Agreed and added in 7a17be172e16a7e1cf40600f855ef61afda6465f.